lib/middleware/karma.js

```diff
@@ -179,6 +179,10 @@ var createKarmaMiddleware = function (
             var filePath = file.path
             var fileExt = path.extname(filePath)
 
+            if (!files.included.hasOwnProperty(i)) {
+              continue
```

Comment:
My original code changed things to a forEach loop, but I though there may be unwanted side effects or opportunities for bugs--for instance, the loop variable `i` is addressed in the loop. So, I kept it simple and just continued the loop on the next item if it was not an own property.